src/plugins/graph/utilities/graph-utils.ts

```diff
@@ -453,7 +480,13 @@ export function setPointCoordinates(props: ISetPointCoordinates) {
         .transition()
         .duration(duration)
         .attr('transform', (aCaseData: CaseData) => {
-          return `translate(${getScreenX(aCaseData.caseID)} ${getScreenY(aCaseData.caseID, aCaseData.plotNum)})`;
+          const x = getScreenX(aCaseData.caseID), y = getScreenY(aCaseData.caseID, aCaseData.plotNum);
+          if (x !== null && isFinite(x) && y !== null && isFinite(y)) {
```

Comment:
In CODAP (in `math-utils.ts`) we've added an `isFiniteNumber()` utility:
```typescript
export function isFiniteNumber(x: any): x is number {
  return x != null && Number.isFinite(x)
}
```
which simplifies and standardizes tests like this.

Reply:
Nice, I copied that into CLUE.